docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+  webapp:
+    image: pathwaycommons/app-ui:latest
+    container_name: appui_webapp
+    ports:
+      - "3000:3000" 
+    environment:
+      ip: "db"      
+    links:
+      - db
+  db:
+    container_name: appui_db
+    image: rethinkdb:2.3.6
+    ports:
+      - "28015:28015"   
```

Comment:
Maybe also the rethink db ui port in case you want to access the web ui for the db?